Updated Rallycross Template

maxmurillo opened the pull request:
Updated the Rallycross template for the new header area option and cleaned up some of the body code

maxmurillo committed:
Created Rallycross Preview Template

Created the file to be able to preview in the rallycross parent category

maxmurillo committed:
Changed Rallycross Header Logo

New Rallycross header logo options for domestic and international

maxmurillo committed:
Updated Rallycross Body

Updated the Rallycross body to remove empty spacing for an empty heading text block

ajrdesign commented on site/snippets/email-header-rallycross.php:
```diff
@@ -0,0 +1,60 @@
+                          </table>
+                        </td>
+
+                      <?php elseif($page->language() == 'uk' || 'au' || 'dk' || 'ch' || 'de' || 'es' || 'fr' || 'it' || 'nl' || 'no' || 'fi' || 'se' || 'jp' || 'kr'): ?>
```
You should only need to do <?php else:> here. You are only checking against one option so all others should be the same.

ajrdesign replied:
You should only need to do <?php else:> here. You are only checking against one option so all others should be the same.

maxmurillo committed:
Cleaned up Rallycross Body

Cleaned up the code for the rallycross body. Also added descriptions for inputs on the builder.
